ansible_collections/arista/avd/roles/eos_designs/python_modules/mlag/avdstructuredconfig.py

```diff
@@ -192,12 +188,10 @@ def vlan_interfaces(self):
         l3_vlan_interface.update(l3_cfg)
 
         # Assembling the interface dict to retain legacy order from Jinja templates.
```

Comment:
@ClausHolbechArista do we need to keep a legacy order while refactoring for 4.0?

Reply:
Not anymore

Reply:
shall I remove this comment?

Reply:
yes remove the comment please

Reply:
we will do it